### PR TITLE
Upload links

### PR DESCRIPTION
**Pull request by C0deJack**
Closing this branch and creating a new branch for refactoring

### CODE REVIEW

File: cms.php [hunk 1]
```diff
@@ -4,23 +4,34 @@
...
     PDO::FETCH_ASSOC);
 
-$query= $db->prepare("SELECT `email`, `phone`, `about`,`deleted`
+$query= $db->prepare("SELECT `email`, `phone`, `about`
 FROM `cms` WHERE `deleted` = 0 ORDER BY `id` DESC LIMIT 1;");
 
-$query1= $db->prepare("SELECT `project_id`, `project_img`, 
-`project_title`,`project_text`, `project_link` FROM `projects` WHERE 
-`project_delete` = 0 ORDER BY `project_id` ASC;");
+$query2= $db->prepare("UPDATE `cms` SET `deleted`= 1;");
```
Comment (nsantiagoblair) [inline cms.php:10]: I think `UPDATE` needs a `WHERE` so it knows what it's updating.

File: cms.php [hunk 2]
```diff
@@ -4,23 +4,34 @@
...
 
+$query3= $db->prepare("INSERT INTO `cms` (`email`, `phone`, `about`)
+VALUES (:email_out, :phone_out, :about_out);");
+
+if (!empty($_POST)) {
```
Comment (nsantiagoblair) [inline cms.php:15]: I would put all the preparing inside the `if` statement too. No need to `prepare()` if you aren't going to `execute()`

File: cms.php [hunk 3]
```diff
@@ -4,23 +4,34 @@
...
+    $query2->execute();
+}
+
+$email_out = $_POST ['email_out'];
+$phone_out = $_POST ['phone_out'];
+$about_out = $_POST ['about_out'];
+
+$query3->bindParam(':email_out', $email_out);
```
Comment (nsantiagoblair) [inline cms.php:23]: you could just stick `$_POST` inside the `bindParam()` to save space.

e.g. `$query3->bindParam(':email_out', $_POST['email_out'])`

File: cms.php [hunk 4]
```diff
@@ -30,40 +41,22 @@
...
-<?php
+<body>
+    <h3>Welcome to the About Me Input Page</h3>
+
+    <nav>
+        <a href="projects.php">Link to Projects Input Page</a><br><br>
+    </nav>
+
+    <form method="post" action="cms.php">
+        <input type="text" value="<?php echo $email; ?>" name="email_out">email
```
Comment (nsantiagoblair) [inline cms.php:52]: Maybe `name` before `value`? It's makes more sense to me since keys become before values `[key => value]`

File: cms.php (hunk 1 above)
Comment (C0deJack) [reply cms.php:10]: This is to set all values in this table to deleted and then a separate query inserts a new 'undeleted' row

File: cms.php (hunk 3 above)
Comment (C0deJack) [reply cms.php:23]: good idea! Thanks

File: cms.php (hunk 4 above)
Comment (C0deJack) [reply cms.php:52]: yep good spot

File: cms.php (hunk 2 above)
Comment (C0deJack) [reply cms.php:15]: You are right, it would be good if the prepare statement was in the if statement but page brings up an error if you do.

**Pull request merged**